### PR TITLE
ACP-3286: Adjusted vertax doc with additional value

### PR DESCRIPTION
**Pull request by vol4onok**
## PR Description
https://spryker.atlassian.net/browse/ACP-3286

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/pbc/all/tax-management/202404.0/base-shop/third-party-integrations/vertex/install-vertex/integrate-the-vertex-app.md [hunk 1]
```diff
@@ -367,35 +367,36 @@ console data:import glossary
...
+| expenses.hash                                                                                | lineItems[].lineItemId                                                   |                                                                                                                                                          |
+| expenses.shipment.shipmentAddress                                                            | lineItems[].customer.destination                                         |                                                                                                                                                          |
+| expenses.shipment.method.shipmentMethodKey                                                   | lineItems[].product.value                                                | Depends on selected shipment method.                                                                                                                     |
```
Comment (matweew) [inline docs/pbc/all/tax-management/202404.0/base-shop/third-party-integrations/vertex/install-vertex/integrate-the-vertex-app.md:394]: Maybe add in the comment that it is inside '<Product>` tag

Comment (vol4onok) [reply docs/pbc/all/tax-management/202404.0/base-shop/third-party-integrations/vertex/install-vertex/integrate-the-vertex-app.md:394]: I didn't get your comment. what's the tag?

Comment (matweew) [reply docs/pbc/all/tax-management/202404.0/base-shop/third-party-integrations/vertex/install-vertex/integrate-the-vertex-app.md:394]: The Tag is in the logs, when you check request logs in the admin panel the logs are in the cml format, so this value will be in `<Product>Bla</Product>` tag.

**Review by matweew — DISMISSED** [review]

File: docs/pbc/all/tax-management/202404.0/base-shop/third-party-integrations/vertex/install-vertex/integrate-the-vertex-app.md (hunk 1 above)
Comment (vol4onok) [reply docs/pbc/all/tax-management/202404.0/base-shop/third-party-integrations/vertex/install-vertex/integrate-the-vertex-app.md:394]: This doc isn't about vertex logs. Logs in vertax have own standart

**Pull request merged**